File: 1-10/day-02/louis/js/main.js
function findNemo(string) {
    index = string.indexOf("Nemo");
    if (index != -1) {
        console.log(`I found Nemo at ${index}`)
    }else{
        console.log("I can't find Nemo")
    }
}

findNemo("I am finding Nemo !");

File: 1-10/day-02/louis/go/main.go
package main

import (
	"fmt"
	"strings"
)

func findNemo(in string) {
	index := strings.Index(in, "Nemo")
	if index != -1 {
		fmt.Printf("I found Nemo at %d", index)
	} else {
		fmt.Printf("I can't find Nemo")
	}

}

func main() {
	findNemo("I am finding Nemo !")
}
